docs/deploy.md

```diff
@@ -0,0 +1,66 @@
+# Deploying the Operator
+
+The following section outlines the steps to deploy application-service on a physical Kubernetes cluster.
+
+## Setting up the AppStudio Build Service environment
```

Comment:
See my other comment, I think this can be removed. It hasn’t been needed for HAS since mid  last year.